lmcache/config.py

```diff
@@ -16,7 +17,18 @@ class LMCacheEngineMetadata:
     """ the format of kv tensors """
     fmt: str
     """ the data type of kv tensors """
-    dtype: str
+    kv_dtype: torch.dtype
+    """ the data type of kv tensors """
+    """ (num_layer, 2, chunk_size, num_kv_head, head_size) """
+    kv_shape: tuple[int, int, int, int, int]
+
+
+@dataclass
+class LMCacheMemPoolMetadata:
+    """ Subset of `LMCacheEngineMetadata` to initialize MemPool"""
+
+    kv_shape: Tuple[int, int, int, int, int]
```

Comment:
What's the meaning of each dimension in this 5-tuple? Let's have a docstring for this.

Also, are we going to support different layouts and shapes? For example, huggingface transformers has a different layout, and seems like when running vLLM on V100 will have a different shape)

Reply:
Will be done in #117